Gamedata/Bluedog_DB/Compatibility/Tweakscale/tweakscale_Engines.cfg

```diff
@@ -44,6 +66,7 @@
         defaultScale = 0.3125
     }
 }
+´
```

Comment:
What is this?

Reply:
@jsolson

Reply:
An apostrophe. Just a stray keypress, it's next to the enter key. It can go.

Reply:
It already was gotten rid of when I split the file.